my-app/src/components/register/register.js

```diff
@@ -43,65 +43,88 @@ addUser = _ => {
   alert("User " + user.username + " was registered");
 }
 
+addUserServer = _ => {
```

Comment:
not sure if you incorporated the password encryption, but when I registered a new user the password wasn't changed when I checked users

Reply:
I'll look into it.

Reply:
Register works just fine, make sure you are running the backend server.